src/background/index.js

```diff
@@ -146,22 +147,33 @@ function sortFunction(a, b) {
 }
 
 async function fetchContestDetails() {
-	const res = await fetch(`https://kontests.net/api/v1/all`, {
+	const res = await fetch(`https://busy-gold-walkingstick-yoke.cyclic.app/api/v1/contests/all`, {
 		method: "GET",
 		headers: {
 			"Content-type": "application/json; charset=UTF-8",
 		},
 	});
+	
 	if (!res.ok) {
 		const message = "An error has occured";
 		throw new Error(message);
 	}
 
 	var contests = await res.json();
+	console.log(contests)
```

Comment:
[FYI] Its good to have dubugging statements like these. But we can avoid them in prod.